src/main/docs/guide/languageSupport/graal/graalServices.adoc

```diff
@@ -30,62 +30,96 @@ $ ./mvnw package -Dpackaging=docker-native
 
 ==== Building a Native Image Without Using Docker
 
-To build your native image without using Docker, install the GraalVM SDK via the https://www.graalvm.org/docs/getting-started/[Getting Started] instructions or using https://sdkman.io/[Sdkman!]:
+To build your native image without using Docker, install a GraalVM JDK. The easiest way to install GraalVM on Linux or Mac is to use https://sdkman.io/[SDKMAN!]. For other installation options, visit the https://www.graalvm.org/downloads/[Downloads page].
 
-.Installing GraalVM {graalVersion} with SDKman
+.Installing GraalVM {graalVersion} with SDKMAN!
 [source,bash,subs="attributes+"]
 ----
-$ sdk install java {graalVersion}.r11-grl
-$ sdk use java {graalVersion}.r11-grl
+$ sdk install java {graalVersion}.-graal
```

Comment:
I couldn't find in src/main/docs where to modify `{graalVersion}`. We should recommend to install the latest LTS release, currently 21.0.5 instead of what is now:
<img width="424" alt="Screenshot 2024-12-14 at 17 31 40" src="https://github.com/user-attachments/assets/70157759-3e22-455d-82ef-330b81e8a333" />

Reply:
it is defined here https://github.com/micronaut-projects/micronaut-core/blob/4.8.x/gradle.properties#L49